explorer/src/bin/admin.ts

```diff
@@ -1,34 +1,8 @@
 import yargs from 'yargs'
-import { Connection } from 'typeorm'
-import { Admin } from '../entity/admin'
-import { createAdmin } from '../support/admin'
-import { closeDbConnection, getDb } from '../database'
+import { seed } from '../cli/admin'
 
-async function bootstrap(cb: any) {
-  const db = await getDb()
-  try {
-    await cb(db)
-  } catch (e) {
-    console.error(e)
-  }
-  try {
-    await closeDbConnection()
-  } catch (e) {
-    console.error(e)
-  }
-}
-
-const seed = async (username: string, password: string) => {
-  return bootstrap(async (db: Connection) => {
-    const admin: Admin = await createAdmin(db, username, password)
-
-    console.log('created new chainlink admin')
-    console.log('username: ', admin.username)
-    console.log('password: ', password)
-  })
-}
-
-const _ = yargs
+/* eslint-disable-next-line no-unused-expressions */
```

Comment:
why do we need this here?

Reply:
Preferred it instead of `const _ = yargs`